pkg/columns/columninfo.go

```diff
@@ -43,11 +57,57 @@ type Column[T any] struct {
 	columnType    reflect.Type // cached type info from reflection
 }
 
+func (ci *Column[T]) getWidthFromType() int {
+	switch ci.kind {
+	case reflect.Uint8:
+		return MaxCharsUint8
+	case reflect.Int8:
+		return MaxCharsInt8
+	case reflect.Uint16:
+		return MaxCharsUint16
+	case reflect.Int16:
+		return MaxCharsInt16
+	case reflect.Uint32:
+		return MaxCharsUint32
+	case reflect.Int32:
+		return MaxCharsInt32
+	case reflect.Uint64, reflect.Uint:
+		return MaxCharsUint64
+	case reflect.Int64, reflect.Int:
+		return MaxCharsInt64
+	case reflect.Bool:
+		return MaxCharsBool
+	}
```

Comment:
You mean change it for readability? Since this is a non-hot code path, I'm happy to do it if it helps - generally (but especially for hotter code) I would prefer the switch as it is much faster.